src/rand.rs

```diff
@@ -327,18 +331,9 @@ mod tests {
             // case in the `SysRandOrDevURandom::Undecided` case. As above, we
             // only do this when there are at least 96 bytes after the first
             // chunk to avoid false positives.
-            if *len > 96 && *len - 96 > max_chunk_len() {
-                assert!(buf[max_chunk_len()..].iter().any(|x| *x != 0));
+            if *len > 256 + 96 {
+                assert!(buf[256..].iter().any(|x| *x != 0));
             }
```

Comment:
I guess this test just doesn't make sense any more, does it? Maybe we should just remove this